rules/test.bzl

```diff
@@ -105,6 +105,7 @@ def _ios_test(name, test_rule, test_suite_rule, apple_library, infoplists_by_bui
             tests.append(test_name)
             split_rule(
                 name = test_name,
+                product_module_name = name.replace("-", "_"),
```

Comment:
Should this use `kwargs.module_name`?

Reply:
Yes! I thought this was a thing but I couldn't find it initially